content/en/docs/tasks/access-application-cluster/service-access-application-cluster.md

```diff
@@ -34,9 +34,9 @@ provides load balancing for an application that has two running instances.
 ## Creating a service for an application running in two pods
 
 1. Run a Hello World application in your cluster:
-
-       kubectl run hello-world --replicas=2 --labels="run=load-balancer-example" --image=gcr.io/google-samples/node-hello:1.0  --port=8080
-
+   ```
```

Comment:
I think if you set the lexer to `shell` or `bash` on these, you will get a little bit of syntax highlighting.

Reply:
Thanks for point that out! I'll set the lexer. Is there a preference between the two? And shall I set the lexer for all shell commands and output whether or not the content gets highlighted. It seems like the right thing to do.

Reply:
I think one is an alias for the other so it probably doesn't matter, but I think I have seen `shell` more often in this repo. I think it's good to set the lexer as a general rule, and you can set it to `none` if you don't want highlighting or if setting the lexer severely breaks the output in code editors like Atom (such as when you have a HEREDOC that injects JSON or YAML, then I know it breaks Atom badly, and the rest of the file will be erroneously highlighted in the editor -- it doesn't seem to affect the output, but it is annoying.